messaging-experiments.json

```diff
@@ -1146,5 +1146,488 @@
         }
       ]
     }
+  },
+  {
+    "id": "MESSAGE_ROUTER_PICTURE_IN_PICTURE_CFR",
+    "enabled": true,
+    "filter_expression": "env.channel == 'release' && env.version >= '78.' && env.version < '79.'",
+    "targeting": "locale == 'en-US' && userPrefs.cfrFeatures &&\n'media.videocontrols.picture-in-picture.enabled'|preferenceValue &&\n[userId, 'messaging-experiments-cfr']|bucketSample(5312, 650, 10000) &&\n((currentDate|date - profileAgeCreated) / 86400000) > 7\n",
+    "arguments": {
+      "slug": "bug-1656382-message-recommend-picture-in-picture-cfr-release-77-80",
```

Comment:
curious how we ensure slug above is same as slug from experimeneter.services.mozilla.com. I see that generated on click of Ready To Ship in Experimenter

Reply:
No, we can't get the actual slug for now. This is merely a placeholder, and we have to update it once it's generated on Experimenter (and yes, that's very error-prone).